### PR TITLE
Pub: v3.7.14, v3.8.14, v3.9.14, and more. [release]

### LINKED ISSUE #144: Please add 3.10.7, 3.9.14, 3.8.14, and 3.7.14
https://pythoninsider.blogspot.com/2022/09/python-releases-3107-3914-3814-and-3714.html

### PR DESCRIPTION
**Pull request by felicianotech**
Closes #144

### CODE REVIEW

File: 3.7/Dockerfile [hunk 1]
```diff
@@ -2,12 +2,13 @@
 
 # Do not edit individual Dockerfiles manually. Instead, please make changes to the Dockerfile.template, which will be used by the build script to generate Dockerfiles.
```
Comment (cclauss) [inline 3.7/Dockerfile:3]: ### Do not edit individual Dockerfiles manually. Instead, please make changes to the Dockerfile.template

Comment (JalexChen) [reply 3.7/Dockerfile:3]: hey @cclauss, what are you referring to specifically in #124?

Comment (cclauss) [reply 3.7/Dockerfile:3]: That it edited individual Dockerfiles.

Comment (JalexChen) [reply 3.7/Dockerfile:3]: i'm not sure which lines you're referring to specifically, but i don't believe these were edited manually. if there's an issue, i'm happy to resolve

Comment (JalexChen) [reply 3.7/Dockerfile:3]: hi @cclauss i'm not sure which lines you're referring to specifically, but i don't believe these were edited manually. if there's an issue, i'm happy to resolve

**Comment by luzfcb** [comment]
@felicianotech @JalexChen v3.9.14 is missing on docker hub
https://hub.docker.com/r/cimg/python/tags?name=3.9.14

**Comment by felicianotech** [comment]
Checking

**Pull request merged**